Test1Rewrite/task2.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <time.h>
+#include <stdlib.h>
+
+bool isSorted(int array[], int size);
+bool shuffle(int array[], int size);
+void swap(int *oneValue, int *anotherValue);
+void monkeySort(int array[], int size);
+bool tests();
+
+int main() {
+    if (!tests()) {
+        return 1;
+    }
+    
+    srand(time(0));
+    
+    int size = 0;
+    printf("Enter a size of the array: ");
+    scanf("%d", &size);
+    while (size <= 0) {
+        printf("Enter a naturalNumber!\n");
+        printf("Enter a size of the array: ");
+        scanf("%d", &size);
+    }
+    int *array = malloc(sizeof(int) * size); 
+    if (array == NULL) {
+        printf("Out of memory\n");
+        return 1;
+    }
+    for (int i = 0; i < size; i++) {
+        printf("Enter the element #%d", i);
+        scanf("%d", &array[i]);
+    }
+
+    printf("Sortedarray: ");
```

Comment:
```suggestion
    printf("Sorted array: ");
```
Только он не sorted, monkeySort не вызывается :)